### PR TITLE
OSDOCS-7755 Adding documentation links to priority ROSA notifications

### PR DESCRIPTION
**Pull request by laubai**
### What type of PR is this?
documentation

### What this PR does / why we need it?
MOBB-F2F: adding documentation URLs and/or more detail for DiskPressure and NetworkUnavailable notifications

### Which Jira/Github issue(s) this PR fixes?
Partial fix for
https://issues.redhat.com/browse/OSDOCS-7755
https://issues.redhat.com/browse/OSDOCS-7756

_Fixes #_

### Pre-checks (if applicable):
- [ ] Tested latest changes against a cluster
- [ ] Included documentation changes with PR
- [ ] If this is a new object that is not intended for the FedRAMP environment (if unsure, please reach out to team FedRAMP), please exclude it with:

    ```yaml
    matchExpressions:
    - key: api.openshift.com/fedramp
      operator: NotIn
      values: ["true"]
    ```

### CODE REVIEW

**Comment by jewzaam** [comment]
/hold

waiting for clarity on text review requirements

**Review by arendej — APPROVED** [review]
lgtm

**Comment by laubai** [comment]
@jewzaam Added the MemoryPressure and PIDPressure information into the notifs.

File: deploy/ocm-agent-operator-managednotifications/node-condition/200-sre-node-condition-managed-notification.ManagedNotification.yaml [hunk 1]
```diff
@@ -11,31 +11,31 @@ spec:
...
+        MemoryPressure condition is true for at least one worker. For more information on memory management in OpenShift applications, see [Understanding managing application memory](https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/cluster_administration/nodes#nodes-cluster-resource-configure-about_nodes-cluster-resource-configure).
       resolvedBody: |-
         MemoryPressure condition is resolved.
       resendWait: 6
     - name: NodeConditionPIDPressureNotification
       severity: Warning
       summary: "Worker node is experiencing PIDPressure"
       activeBody: |-
-        PIDPressure condition is true for at least one worker.
+        PIDPressure condition is true for at least one worker. This means that more than the maximum allowed number of processes are present on a worker node. This is likely to create instability in the node, as required system services may not be able to create new processes. Reduce the number of pods on the node, or the number of processes generated by applications on the node.
```
Comment (jewzaam) [inline deploy/ocm-agent-operator-managednotifications/node-condition/200-sre-node-condition-managed-notification.ManagedNotification.yaml:30]:
```suggestion
        PIDPressure condition is true for at least one worker. This means that more than the maximum allowed number of process identifiers are consumed on one or more worker nodes. This is likely to create instability in the node, as required system services may not be able to create new processes. Reduce the number of pods scheduled to the impacted nodes, or reduce the number of process identifiers consumed by applications on the impacted nodes.
```

No KCS on what PIDs are and how they're consumed?

**Review by jewzaam — CHANGES_REQUESTED** [review]

**Review by jewzaam — APPROVED** [review]

**Review by bchandra-ocp — APPROVED** [review]
thanks

**Pull request merged**